tests/test_TFNetworkLayer.py

```diff
@@ -868,6 +868,54 @@ def test_pool_layer_NCHW():
     print(seq_lens)
 
 
+def test_batch_norm():
+  with make_scope() as session:
+    import numpy as np
+    from tensorflow.python.client import timeline
+    net = TFNetwork(extern_data=ExternData())
+    net.train_flag = True
+    with tf.variable_scope("src_nchw"):
+      src_nhwc = InternalLayer(name="src_nchw", network=net, out_type={"dim": 16,
+                                                                       "shape": (16, None, 16),
+                                                                       "batch_dim_axis": 0,
+                                                                       "time_dim_axis": 2,
+                                                                       "feature_dim_axis": 1,
+                                                                       "sparse": False
+                                                                       })
+      src_nhwc.output.placeholder = tf.placeholder(shape=(None, 16, None, 16), dtype=tf.float32)
+      src_nhwc.output.size_placeholder = {1: tf.placeholder(shape=(None,), dtype=tf.int32)}
+
+    with tf.variable_scope("batch_norm_masked_nchw"):
+      batch_norm_1 = BatchNormLayer(name="batch_norm_masked_nchw", network=net, masked_time=False,
```

Comment:
You write `batch_norm_masked_nchw` but you have `masked_time=False`. This seems wrong...
(Please double check your own code...)